### PR TITLE
Standardise styling for UG

### PR DESCRIPTION
**Pull request by qianz-z**

### CODE REVIEW

**Comment by Than-Duc-Huy** [comment]
In the Command Summary, do you want to separate GUI and CLI?

**Comment by qianz-z** [comment]
> te

I think there is no need since most of the functions uses the same syntax, and only edit has 2 different format for cli and gui respectively.

**Comment by Than-Duc-Huy** [comment]
ok can

File: docs/UserGuide.md [hunk 1]
```diff
@@ -189,21 +189,21 @@ Format: `/edit RECIPE_INDEX COMMAND_FLAG RECIPE_FLAG PARAMETERS`
     - Exactly **1 (one)** integer must be provided.
   - Example:
   ```
-  /edit 2 -del -i 2 
-      
-  Delete "Lettuce"
```
[the remark this answers is not in the document]
Comment (qianz-z) [reply docs/UserGuide.md:194]: I think some of the outputs are wrongly reflected, and some of them are quite lengthy so it is unreadable to show all the outputs.

**Pull request merged**